pkg/appgateserver/jsonrpc.go

```diff
@@ -13,6 +14,15 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// InterimJSONRPCRequestPayload is a partial JSON RPC request payload that
+// excludes the params field, which is unmarshaled sperately.
```

Comment:
```suggestion
// excludes the params field, which is unmarshaled separately.
// This is used because the proto object cannot be unmarshalled directly, so this is a
// temporary object to be used interim.
```